lambdanotes/src/components/Edit.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { updateNote } from '../actions';
+import Editview from './Editview';
+
+
+class Edit extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: '',
+            textbody: ''
+            
+        }
+    }
+
+    handleChange = e => {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+    
+    
+    updateNote = e => {
+        e.preventDefault();
+        let updatedNote = {
+            id: this.props.match.params.id,
+            title: this.state.title,
+            textbody: this.state.textbody
+        }
+        this.props.updateNote(updatedNote);
+        this.props.history.push("/")
+        window.location.reload();
```

Comment:
A hack that certainly works, but is not very React-like. What are some other ways to address this? To you have a `get` method somewhere that you could call instead? Refactor this a bit to see what you come up with, but this hack should not be used going forward.